pkgs/development/libraries/libtiff/libtiff_t.nix

```
@@ -0,0 +1,73 @@
+{ lib
+, stdenv
+, fetchzip
+
+, autoreconfHook
+, pkg-config
+, sphinx
+
+, libdeflate
+, libjpeg
+, xz
+, zlib
+}:
+
+# This is a fork created by the hylafaxplus developer to
+# restore tools dropped by original libtiff in version 4.6.0.
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "libtiff";
```

Maybe change pname as well ?

```suggestion
  pname = "libtiff_t";
```

---

I lean towards no: `libitff_t` is a purely "nixpkgs-internal" name.  We have to use it as we have to use a unique name within all packages.  This `pname` is not needed to be unique and can be anything.  It should be as close to the upstream name as possible (the attribute name also, but it can't be used twice).

Unfortunatelly, I haven't see any -- more or less official -- name of the forked library ... up to `libitff`, which is even the second-level domain.

This is not a strong opinion, changing this certainly wouldn't do any technical harm.  We do you think?

---

`pname` is part of the storage path and having it unique can help with debugging. You can immediately see if storage path belongs to original `libtiff` or to fork. It can be very confusing if your closure depends on both - it can look as a bug without knowing the details.

---

Feel free do decide as you want.

---

You're right, ease of debugging is more important than an official package name.

Branches updated (and rebased again).

---

Thanks.